packages/api/app/Validators/UpdateUser.js

```diff
@@ -6,6 +6,17 @@ class UpdateUser extends BaseValidator {
 
 		return {
 			email: `unique:users,email,id,${userId}`,
+			zipcode: 'number',
+			cpf: 'number',
```

Comment:
É muito melhor trabalhar com CPF em string do que em números.

Se CPF for número não haverá zeros a esquerda. ex. 054.545.454-43 será 54.545.454-43. É por isso que muitos sistemas "comem" o zero do CPF rsrs.

Reply:
@nicholasio se você observar na migration os campos estão todos como string, apenas nos validators eu fiz isso. Isso serve para que eu envie um número e não uma string de fato. Na prática você precisa enviar em cpf algo do tipo "03089089011", isso evita que você mande tipo "nichollas123". Ou seja, assim eu garanto que seja enviado pelo front um número. Isso é tão verdade que tanto no faker quanto no teste eu uso string numéricas. Essa explicação é válida para os demais.

Reply:
acho que seria interessante sim validar o CPF, é coisa simples de adicionar no back.

Reply:
No último commit eu adicionei a validação do cpf.